packages/dpos/store/actions/voting.js

```diff
@@ -77,20 +77,20 @@ export const votesSubmitted = ({ fee, votes }) =>
   async (dispatch, getState) => {
     const state = getState();
     // @todo Fix this by #3898
-    const activeAccount = {
-      ...state.account.info.LSK,
-      hwInfo: isEmpty(state.account.hwInfo) ? undefined : state.account.hwInfo,
-      passphrase: state.account.passphrase,
+    const activeWallet = {
+      ...state.wallet.info.LSK,
```

Comment:
do we have multiple token on the state?
we need to avoid any hardcoded key/value